.editorconfig

```diff
@@ -0,0 +1,10 @@
+[*]
```

Comment:
Please delete this file as I think it's editor specific?

Reply:
Will do